lesson18/style.css

```diff
@@ -0,0 +1,134 @@
+body,html{
+    font-family: YuGothic,'Yu Gothic','Yu Gothic UI','ＭＳ ゴシック','ヒラギノ角ゴシック','Hiragino Sans','Hiragino Kaku Gothic ProN','ヒラギノ角ゴ ProN W3', sans-serif;
+    font-size: 16px;
+    color:rgb(70, 70, 70);
+    line-height: 1.2rem;
+    letter-spacing: 0.05rem;
+}
+a{
+    text-decoration: none;
+}
+a:hover{
+    opacity: 0.7;
+    transition: 0.2s;
+}
+h1{
+    font-size: 2rem;
+}
+p{
+    margin:15px 0px;
+}
+/*ローディング*/
+.loading{
+    text-align: center;
+    background: #fff;
+    margin-top:50px;
+}
+
+/*スライダー*/
+.slider{
+    position: relative;
+    text-align: center;
+    height: 80vh;
+    width: auto;
+    height: 440px;
+    max-width: 600px;
+    margin: auto;
+    padding:50px;
+}
+.slider__item{
+    position: absolute;
+    top: 70px;
+    right: 0;
+    left: 0;
+    text-align: center;
+    z-index: 0;
+    padding: 50px;
+}
+.slider__item.is-active{
+    z-index: 1;
+}
+.slide-image{
+    width: 100%;
+}
+@media(max-width:414px){
+    .slider{
+        width: auto;
+        height: 290px;
+    }
+}
+/*ボタン*/
+.slider-btn{
+    position: absolute;
+    font-size: 1rem;
+    color:#5d5d5d;
+    top: calc(50% + 50px);
+    background: #fff;
+    border: 2px solid #5b5b5b;
+    padding: 8px 10px;
+    border-radius: 50px;
+    z-index: 999;
+    opacity:0.9;
+}
+.slider-btn:hover{
+    opacity: 0.6;
+}
+.slider-btn:disabled{
+    pointer-events: none;
+}
```

Comment:
IMO
I thought it would be better if I could clearly see 'disabled' when it is 'disabled'.

I also thought it should be `cursor: pointer;` to convey that dot pagination can also be pressed.

Reply:
Thank you for your　comment!
Certainly, as you say, it's very important to be able to clearly determine if you can push.
Thank you for teaching me!

I fixed CSS for disabled and dot-pagination.
https://github.com/nakagawamai/js-lesson/pull/20/commits/4ce8dafdd4f3901414f0d64247c4d5b76a4d1d33